symfony/3-nginx/etc/confd/templates/nginx/nginx_site.conf.tmpl

```diff
@@ -0,0 +1,27 @@
+server {
+    listen 80;
+
+    root /app/web;
+
+    server_name localhost;
+
+    location / {
+        try_files $uri /app.php$is_args$args;
+    }
+
+    location ~ ^/(app|app_dev|config)\.php(/|$) {
```

Comment:
I'd be ok with just the image replacing https://github.com/continuouspipe/dockerfiles/blob/master/php-nginx/etc/confd/templates/nginx/site_phpfpm.conf.tmpl with the updated location block short term if a decision can't be made. At least the duplication would only be contained to the base image and not projects that use it